script/Ciel/Day 14/14_01B_02_CIEL14B_3.txt

```diff
@@ -631,22 +721,30 @@ So what?"
 [sha:99dd331ad7f6201a5bac12262c8e63a01bb33674]{
 -- Page 128, Offset 39783.
 -- 『愚問だったようだ。
-"It was foolish of me to point that out.
+%{i}
+"It appears I needn't have mentioned it.
+%{/i}
```

Comment:
I think it should be a bit smarmier, because this really is one of Roa's best quips of all time
>Sorry, stupid of me to mention it.

Reply:
Tough line, but not sure that really comes off a smarmy to me. The VA might help, but may need something else just to make it clear.

Reply:
Hmmm yeah, mine isn't much of an improvement at the end of the day.
I feel like there is a snappy version out there, but I can't quite grasp it.